src/components/fontawesome.tsx

```diff
@@ -54,5 +55,6 @@ library.add(
   faUtensils,
   faBed,
   faShoppingCart,
-  faBinoculars
+  faBinoculars,
+  faExpand
```

Comment:
https://fontawesome.com/icons/external-link-alt?style=solid
As suggested by @shumalll, let's use the `external-link-alt` icon, I'm so sorry I forgot to update the mocks after our review.

Reply:
Oh ya my bad, I forgot about that too. I will update that haha